tests/unit/xoopsLib/Xoops/Core/Database/ConnectionTest.php

```diff
@@ -15,137 +15,135 @@ class ConnectionTest extends \PHPUnit_Framework_TestCase
 {
     protected $myclass = '\Xoops\Core\Database\Connection';
 
+    /** @var  Xoops\Core\Database\Connection */
+    protected $object;
+
     public function setUp()
-	{
+    {
+        $params = array();
+        $config = new Configuration();
+        $eventManager = new EventManager();
+        $driver = new Doctrine\DBAL\Driver\PDOMySql\Driver();
+
+        $this->object = new $this->myclass($params, $driver, $config, $eventManager);
     }
 
     public function test___construct()
-	{
-		$params = array();
-		$config = new Configuration();
-		$eventManager = new EventManager();
-		$driver = new Doctrine\DBAL\Driver\PDOMySql\Driver();
-
-		$instance = new $this->myclass($params, $driver, $config, $eventManager);
-		$this->assertInstanceOf('\Xoops\Core\Database\Connection', $instance);
-    }
-
-	public function test_setSafe()
-	{
-		$class = $this->myclass;
-		$class::setSafe(true);
-		$x = $class::getSafe();
-		$this->assertTrue($x);
-
-		$class::setSafe(0);  // arg is not boolean => ignore
-		$x = $class::getSafe();
-		$this->assertTrue($x);
-
-		$class::setSafe(false);
-		$x = $class::getSafe();
-		$this->assertFalse($x);
-	}
-
-	public function test_getSafe()
-	{
-	}
-
-	public function test_setForce()
-	{
-		$class = $this->myclass;
-		$class::setForce(true);
-		$x = $class::getForce();
-		$this->assertTrue($x);
-
-		$class::setForce(0);  // arg is not boolean => ignore
-		$x = $class::getForce();
-		$this->assertTrue($x);
-
-		$class::setForce(false);
-		$x = $class::getForce();
-		$this->assertFalse($x);
-	}
-
-	public function test_getForce()
-	{
-	}
-
-	public function test_prefix()
-	{
-		$class = $this->myclass;
-		$x = $class::prefix('');
+    {
+        $params = array();
+        $config = new Configuration();
+        $eventManager = new EventManager();
+        $driver = new Doctrine\DBAL\Driver\PDOMySql\Driver();
+
+        $instance = new $this->myclass($params, $driver, $config, $eventManager);
+        $this->assertInstanceOf('\Xoops\Core\Database\Connection', $instance);
+    }
+
+    public function test_setSafe()
+    {
+        $class = $this->myclass;
+        $this->object->setSafe(true);
+        $x = $this->object->getSafe();
+        $this->assertTrue($x);
+
+        $this->object->setSafe(false);
+        $x = $this->object->getSafe();
+        $this->assertFalse($x);
+    }
+
+    public function test_getSafe()
+    {
+    }
+
+    public function test_setForce()
+    {
+        $this->object->setForce(true);
+        $x = $this->object->getForce();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $x. Configured minimum length is 3.](https://www.codacy.com/app/richard_3/XoopsCore/file/2389370479/issues/source?bid=3194562&fileBranchId=3229089#l61)